OpenEphys.Onix/OpenEphys.Onix/NeuropixelsV1eMetadata.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace OpenEphys.Onix
+{
+    class NeuropixelsV1eMetadata
+    {
+        const uint OFFSET_ID = 0;
+        const uint OFFSET_VERSION = 10;
+        const uint OFFSET_REVISION = 11;
+        const uint OFFSET_FLEXPN = 20;
+        const uint OFFSET_PROBEPN = 40;
+
+        public NeuropixelsV1eMetadata(I2CRegisterContext serializer)
+        {
+            var flexI2C = new I2CRegisterContext(serializer, NeuropixelsV2e.FlexEEPROMAddress);
```

Comment:
```suggestion
            var flexI2C = new I2CRegisterContext(serializer, NeuropixelsV1e.FlexEEPROMAddress);
```

Assumimg this should be using a reference to the V1 constants rather than V2?